sdk/go/common/util/gitutil/git.go

```diff
@@ -346,12 +346,12 @@ func getAuthForURL(url string) (string, transport.AuthMethod, error) {
 	}
 	// If we have no auth, try to get it from the environment.
 	if auth == nil {
-		if strings.Contains(endpoint, GitHubHostName) && os.Getenv("GITHUB_TOKEN") != "" {
+		if (strings.Contains(endpoint, GitHubHostName) || strings.Contains(endpoint, "github")) && os.Getenv("GITHUB_TOKEN") != "" {
```

Comment:
You can drop the `strings.Contains(endpoint, GitHubHostName)`, since if the second bit matches the first one also will.

```suggestion
		if strings.Contains(endpoint, "github") && os.Getenv("GITHUB_TOKEN") != "" {
```